ev_charge/lib/core/database.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:drift/drift.dart';
+import 'package:drift_flutter/drift_flutter.dart';
+import 'package:path_provider/path_provider.dart';
+
+part 'database.g.dart';
+
+class EVCarModels extends Table {
+  late final id = integer().autoIncrement()();
+  late final modelName = text().withLength(min: 3, max: 64)();
+  late final modelYear = integer()();
+  late final batteryCapacity = real()();
+  late final maxChargingPower = real()();
+}
+
+class UserEVs extends Table {
+  late final id = integer().autoIncrement()();
+  late final carModelId = integer().references(EVCarModels, #id)();
+  late final userSetName = text().withLength(min: 3, max: 64)();
+  late final currentCharge = real()();
+}
+
+class Constraints extends Table {
+  late final id = integer().autoIncrement()();
+  late final userCarModelId = integer().references(UserEVs, #id)();
+
+
+  // The day of the week, 0 is Monday, 6 is Sunday
+  late final Column<int> dayOfWeek = integer().check(dayOfWeek.isBetweenValues(0, 6))();
+
+  // Stored as minutes since midnight: 0 = 00:00, 60 = 01:00, ..., 1380 = 23:00
+  // Displayed in UI through TimeOfDay.fromMinutes(startMinutes)
+  // Might be changed depending on back-end requirements
+  late final Column<int> startMinutes = integer().check(startMinutes.isBetweenValues(0, 1440))();
+  late final Column<int> endMinutes = integer().check(endMinutes.isBetweenValues(0, 1440))();
+
+  // Optional: Prevent inverted ranges
+  @override
+  List<String> get customConstraints => [
+    'CHECK (start_minutes < end_minutes)'
+  ];
+}
+
+class Schedule extends Table {
```

Comment:
```suggestion
class Schedules extends Table {
```

drift docs siger at det generer en Schedule row klasse ud fra denne table så plural giver mere mening